tw-tasks-jobs/src/main/java/com/transferwise/tasks/impl/jobs/JobsProperties.java

```diff
@@ -13,4 +13,6 @@ public class JobsProperties {
   private String processingBucket = IBucketsManager.DEFAULT_ID;
 
   private boolean testMode = false;
+
+  private boolean autoStartProcessing = true;
```

Comment:
Currently this will only disable initialisation of jobs, `autoStartProcessing` would also need to be set to false for tasks to disable the consuming of triggered jobs.

Wonder if we should make the handler skip triggered jobs also if this is set or rename this to something along the lines of `autoInitialize` 🤔